sdktests/testapi_sdk_client.go

```diff
@@ -77,7 +77,14 @@ func WithEventsConfig(eventsConfig servicedef.SDKConfigEventParams) SDKConfigure
 // WithPollingConfig is used with StartSDKClient to specify a non-default polling configuration.
 func WithPollingConfig(pollingConfig servicedef.SDKConfigPollingParams) SDKConfigurer {
 	return helpers.ConfigOptionFunc[servicedef.SDKConfigParams](func(configOut *servicedef.SDKConfigParams) error {
-		configOut.Polling = o.Some(pollingConfig)
+		dataSystem := configOut.DataSystem.OrElse(servicedef.DataSystem{})
```

Comment:
For FDv1 support, it is assumed that there is only ever polling or streaming. So we can safely assume, for this first iteration, that these must be equivalent to the primary synchronizer of FDV2 fame. 

Subsequent commits will rename these methods to be like `WithPrimaryPolling` or something of that nature. Letting that evolve as I refactor.